navsat_transform:
  frequency: 0
  delay: 0.0
  magnetic_declination_radians: 38.34774
  yaw_offset: 0.00000
  zero_altitude: false
  broadcast_utm_transform: false
  publish_filtered_gps: false
  use_odometry_yaw: false
  wait_for_datum: false
